### DISCUSSION TITLE
Switch OFF Logging

### DISCUSSION THREAD

**Discussion by HiQual** [john30/ebusd#856] [Q&A] [answered]
Hello.
Can I switch OFF ebusd logging (to /var/log/ebusd.log).
It's been up and running fine for months so I don't want the constant writing to my sdhc card.
Many thanks.

**Comment by cyberthom42** [comment, marked as answer]
What about the options:
`--loglevel=error`
to reduce or
`--logfile=/dev/null`

**Reply by HiQual** [reply]
Perfect - Many thanks.
